### ISSUE TITLE
Your connection is not secure on valet domains

### ISSUE THREAD

**Issue by anuj9196** [jmarcher/valet-linux#46]
I'm using valet v2.0.23 on ubuntu 16.04.

earlier everything was working fine. but now no URL is being accessed. every URL is redirecting to `https` and gives browser error `Your connection is not secure`.

How to serve domains of `valet paths` to use `https://`?

Why are all .dev domains redirecting to `https://` and being blocked by the browser?

**Comment by robert-moore96** [comment]
https://laravel-news.com/chrome-63-now-forces-dev-domains-https

**Comment by anuj9196** [comment]
But I'm using `Firefox Developer Edition 59.0b7 (64-bit)`

**Comment by anuj9196** [comment]
changed `.dev` to something else using
```
valet domain tld-name
```
and it is working now

**Comment by jmarcher** [comment]
A few things:

1. The certificate that is creates is self-signed, that means almost any browser will not detect it as secure
2. If you completed any request under https your browser will always try to redirect you to https it has nothing to do with valet it self.
3. Although this package may still work (did not updated it for almost a year) I deprecated it and my merged my code into cpriego/valet-linux, it was a Ubuntu only package at the beginning I ported the code to work on any Linux distribution but now I lack the time to update this repository therefore I recommend anyone using my package to upgrade to cpriego/valet-linux I'm using it myself on Manjaro Linux
